src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/PetTsvCreator.java

```diff
@@ -20,6 +20,8 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import static org.broadinstitute.hellbender.tools.variantdb.nextgen.CreateVariantIngestFiles.isNoCall;
```

Comment:
i'm a little surprised you needed this import. I think it's common when calling static methods on other classes to actually use the class name:
CreateVariantIngestFiles.isNoCall(variant)
so maybe just import the class as a regular import (instead of a static import of the method)